package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sphinx_rtd_theme",
-  "version": "0.5.0rc1",
+  "version": "0.5.0rc2",
```

Comment:
This seems like it likely shouldn't be here?

Reply:
It should be, it was missed on last release. `setup.cfg` and `package.json` match.

Reply:
Added #940 to catch this next release